### PR TITLE
Add how to test your simple shell

### PR DESCRIPTION
**Pull request by bravemaster3**
* Added a testing process in the testing.md file
* Linked the testing.md in the README.md of the simple-shell directory.

### CODE REVIEW

**Comment by bravemaster3** [comment]
I added some guidelines for testing a simple shell...
Now, I have one concern about the structure in the table of the README.md

| File/folder    | Description    | External resource |

What is that External resource supposed to mean?
Because it is useless to always have to link to alx. What did you intend for that column?

**Comment by geoffreylgv** [comment]
> I added some guidelines for testing a simple shell... Now, I have one concern about the structure in the table of the README.md
> 
> | File/folder | Description | External resource |
> 
> What is that External resource supposed to mean? Because it is useless to always have to link to alx. What did you intend for that column?

The external resource is for a breve explanation or overview, we can't link to alx (intranet concept) 'cause if we do that we are in violation of the alx's code of conduct. So what we do is link a good resource of overview referring to the specific topic of resource we are adding (note that it's optional)

**Review by geoffreylgv — COMMENTED** [review]
Really important this part, I mean all the testing part.
Thank you

File: simple-shell/README.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ Comprehensive collection and resources of learning and implement the simple shel
 | File/folder    | Description    | External resource |
 | :--- | :--- | :--- |
 |[learning-material.md](learning-material.md)| Links to blogs/articles to implement your own Linux Bourne Shell |[Alx & Holberton School: software engineering programme](alxafrica.com) |
+|[testing.md](testing.md)| How to properly test your code to avoid last minute bad surprises |[Alx & Holberton School: software engineering programme](alxafrica.com) |
```
Comment (geoffreylgv) [inline simple-shell/README.md]: Ok, for external resources, 

- we should add a link containing an explanation on how to test efficaciously the simple shell or the learning material we've added. 
- Also we can add an external link to maybe a GitHub page that explains how to test it in this case.
- Another thing, in the way the external link column is optional, we are not obliged to add it, we can leave it blank.

**Comment by bravemaster3** [comment]
Thanks!

> > I added some guidelines for testing a simple shell... Now, I have one concern about the structure in the table of the README.md
> > | File/folder | Description | External resource |
> > What is that External resource supposed to mean? Because it is useless to always have to link to alx. What did you intend for that column?
> 
> The external resource is for a breve explanation or overview, we can't link to alx (intranet concept) 'cause if we do that we are in violation of the alx's code of conduct. So what we do is link a good resource of overview referring to the specific topic of resource we are adding (note that it's optional)

Clear now. Thanks!

**Review by geoffreylgv — APPROVED** [review]
Good, thanks for this resource @bravemaster3 👍🏾

**Pull request merged**